### DISCUSSION TITLE
execute flutter build aar, fail

### DISCUSSION THREAD

**Discussion by wlong800** [firebase/flutterfire#3685] [Q&A]
describe:
update firebase_auth : 0.18.0+1， then execute flutter build aar,  report an error。 before use firebase_auth : 0.16.0 is No problem
I hope 0.18.0+1 can execute flutter build aar, thinks

my question:
FAILURE: Build failed with an exception.

What went wrong:
Execution failed for task ':firebase_analytics:uploadArchives'.
Could not publish configuration 'archives'
Could not write to file '/Users/wanglong/.pub-cache/hosted/mirrors.tuna.tsinghua.edu.cn%47dart-pub/firebase_analytics-6.0.0/android/build/poms/pom-default.xml'.
Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

Get more help at https://help.gradle.org

BUILD FAILED in 2s

flutter doctor:
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.20.4, on Mac OS X 10.14.6 18G103, locale
    zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✗] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS
      development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin
        code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install:
        sudo gem install cocoapods
[✓] Android Studio (version 4.0)
[✓] Connected device (1 available)

! Doctor found issues in 1 category.

**Comment by evan505** [comment]
same problem +1

**Comment by chandragupt1201** [comment]
Did anyone find solution for this issue ?
